Signal/src/ViewControllers/MessageMetadataViewController.swift

```diff
@@ -4,6 +4,12 @@
 
 import Foundation
 
+@objc
+enum MessageMetadataViewMode: UInt {
+    case focusOnMessage
+    case focusOnMetadata
+}
+
 class MessageMetadataViewController: OWSViewController {
```

Comment:
I propose that we rename `MessageMetadataViewController` as `SingleMessageViewController`.

Reply:
:+1: Or maybe `MessageDetailViewController` or `MessageInfoViewController`

Reply:
MessageDetailViewController